pom.xml

```diff
@@ -58,7 +58,7 @@
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <cdap.version>6.4.0-SNAPSHOT</cdap.version>
-    <debezium.version>0.9.5.Final</debezium.version>
+    <debezium.version>1.3.1.Final</debezium.version>
```

Comment:
why are we defining the verion of debezium at delta repo ? That is source plugin specific , right ?

Reply:
yeah thats right. Let's remove it.

Reply:
Its unused